src/components/pages/packages/flyouts/packageNew/packageNew.js

```diff
@@ -52,23 +55,40 @@ export class PackageNew extends LinkedComponent {
   apply = (event) => {
     event.preventDefault();
     const { createPackage } = this.props;
-    const { type, packageFile } = this.state;
+    const { packageType, configType, customConfigName, packageFile } = this.state;
+
+    // If configType is 'Custom' concatenate 'Custom' with customConfigName.
+    let configName = '';
```

Comment:
This feels like the backend or reducer should do this.

Reply:
Not necessary. We can talk about this. I thinks it's a personal preference.